.gitattributes

```diff
@@ -1,2 +1,2 @@
 # Some storm-webapp logviewer tests require input files to have LF line endings due to byte counting.
-storm-webapp/src/test/resources/*.log.test text eol=lf
\ No newline at end of file
+* text=auto
```

Comment:
I agree that it makes sense to force LF on check in, but we should maybe ensure some files are excluded, e.g. png and svgs and other image files. We have a couple in the docs.

Reply:
`text` automatically recognizes text formats (according to https://git-scm.com/docs/gitattributes) and I doubt that `git` handles this badly. PNG is binary and SVG is text-based, so afaik PNG would be excluded automatically and SVG's EOL whould be converted to LF on check-in which makes sense. The conversion should be done once manually with `git rm -r --cached && git add .`.